### PR TITLE
修改readme

### PR DESCRIPTION
**Pull request by longyi365**
**I rearranged the readme file**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,37 +25,26 @@
     用户密码：123123
 
 
-# sfa_vue
+### 安装项目
```
Comment (wuyanshen) [inline README.md]: very good suggest!

Comment (longyi365) [reply README.md]: thank you

**Comment by longyi365** [comment]
Thank you

**Comment by wuyanshen** [comment]
readme is fix successful

**Pull request merged**